rebasehelper/checkers/rpmdiff_tool.py

```diff
@@ -151,11 +151,34 @@ def run_check(cls, results_dir, **kwargs):
                 lines.append('Following files were %s:' % key)
                 lines.extend(val)
 
-        rpmdiff_report = os.path.join(cls.results_dir, 'report-' + cls.CMD + '.log')
+        rpmdiff_report = os.path.join(cls.results_dir, cls.CMD + '.txt')
```

Comment:
I think `checkers/rpmdiff/report.txt` is better than `checkers/rpmdiff/rpmdiff.txt`.